src/scheduler.rs

```diff
@@ -17,6 +20,11 @@ pub async fn run_scheduler() -> Result<(), JobSchedulerError> {
             println!("Executing job at {:?}", Utc::now());
 
             let check_result = job_checker.check_url("https://sentry.io").await;
+            // TODO: Get this from configuration.
+            // TODO: Producer should be instantiated with these values and shared
+            let config = KafkaConfig::new_config(["0.0.0.0".to_string()].to_vec(), None);
+            let topic = Topic::new("uptime-checker-results");
+            let _ = produce_checker_result(&check_result, topic, config).await;
```

Comment:
nice, let's move the config outside of the function though I think.

If you address the earlier comment of making a ResultsProducer struct we can pull that out also (will need to wrap in Arc)


Once #43 is merged we can pass the config into `run_scheduler` and it can construct the KafkaConfig from that